rails/config/locales/sw/dashboard.sw.yml

```
@@ -0,0 +1,53 @@
+en:
```

The file name has been updated to use `.sw` but the translation file key is still keyed to english translations. Let's update the file to denote "sw":

```suggestion
sw:
```

Even if the translations in the file are still English, the top-level key dictates the locale lookup and must be updated to `sw`.

---

I am working on it and will update the PR once I have completed the translation

---

I have translated and updated the `sw.yml` file